pageserver/src/tenant.rs

```diff
@@ -1102,9 +1103,9 @@ impl Tenant {
     /// Subroutine of `load_tenant`, to load an individual timeline
     ///
     /// NB: The parent is assumed to be already loaded!
-    #[instrument(skip_all, fields(timeline_id))]
+    #[instrument(skip(self, local_metadata, ctx))]
```

Comment:
I think I fixed this already?
```suggestion
    #[instrument(skip_all, fields(timeline_id=%timeline_id)]
```

I'll be adding parameters to this, skip_all I think is more ergonomic?

Reply:
Per my testing skip_all skips everything including explicitly mentioned fields, that was quite surprising

Reply:
Ah, it's possible that you've been paying more attention :) Oki, then it's fine. I'll resolve.